ui/shared/components/page/PageHeader.jsx

```diff
@@ -78,7 +78,7 @@ export default () => (
     {NO_HEADER_PAGES.map(noHeaderRoute)}
     {SIMPLE_HEADER_PAGES.map(simpleHeaderRoute)}
     <Route path="/project/:projectGuid/saved_variants/:variantPage?/:breadcrumbId?/:tag?" component={ProjectSavedVariantsPageHeader} />
-    <Route path="/project/:projectGuid/:breadcrumb/:breadcrumbId?/:breadcrumbIdSection*" component={ProjectPageHeader} />
+    <Route path="/project/:projectGuid/:breadcrumb/:breadcrumbId?/:breadcrumbIdSection/:breadcrumbIdSubsection*" component={ProjectPageHeader} />
```

Comment:
`:breadcrumbIdSection` needs to be optional (`?`), going to the main project page should be broken with this change. Please make sure to test that all the existing projects pages still look right after this change